recipes/paramspace/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "paramspace" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5a8d171663cfca822bef5688c667f6392f63440b2ba5294be25b0f408654f051
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy >=1.17.3
+    - ruamel.yaml >=0.16.5
+    - xarray >=0.10.9
```

Comment:
Do you need all of these at install time?

Reply:
I don't! Thanks for bringing this up; 6b45cee removes them.